src/testing/factories/modelVersion.ts

```diff
@@ -37,7 +40,7 @@ export const MockModelVersion1Model1: ModelVersion = {
   status: ModelVersionStatus.Released,
   applications: [],
   metadata: {},
-  isExternal: false,
+  isExternal: true,
```

Comment:
It's a little bit alternative logic, isExternal: true but the same time  isReleasedAndInternal: true